src/server.rs

```diff
@@ -510,13 +509,20 @@ impl<S : CallStarter> httpbis::Service for GrpcHttpService<S> {
                             ))
                     }
                 })
+                .normalize_metadata(
```

Comment:
I added the 's3' stream back in order to terminate the HTTP response correctly if the grpc_response does not contain trailing metadata.